.github/workflows/ci.yml

```diff
@@ -1,7 +1,7 @@
 name: ci
 
 env:
-  GO_VERSION: 1.19.3
+  GO_VERSION: 1.20
```

Comment:
Looks like the `verify-go-modules` action failed because YAML trims this down to "1.2". 
Try wrapping it in a single quote?